src/main/java/com/rtb/projectmanagementtool/user/UserData.java

```diff
@@ -0,0 +1,79 @@
+package com.rtb.projectmanagementtool.user;
+
+import java.util.HashSet;
+
+/** Enum containing skills for user. */
+enum Skills {
+  //not sure what to add here
```

Comment:
Do I leave it blank or create an empty HashMap<String>? I've never used enum and am a bit unclear.

Reply:
enums doc:  https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Specifically look at how they build the enum "class" for planets.  that'll allow you to specify a string description for your enum values.